### PR TITLE
Added rough better description

### PR DESCRIPTION
**Pull request by mraaroncruz**
This is rough. Suggestions welcome.

### CODE REVIEW

**Comment by DominikGuzei** [comment]
could you post a screenshot :-) text looks good!

File: source/index.html.erb [hunk 1]
```diff
@@ -14,9 +14,31 @@ title: Foodhack Vienna | Coming Spring 2015
...
-  <p>
-    Foodhack Vienna is a 2 day contest where people hack on software to help change Austria's food industry. Our main goal is to significantly reduce the amount of food waste that Austrians produce.
-  </p>
+  <div class="wrapper">
+    <div class="explanation">
+      <h2 class="headline">What is Foodhack</h2>
+      <p>Foodhack is a 2 day contest where developers, designers and UX experts come together and build software to help fix Austria's food waste problem.</p>
+    </div>
+    <div class="explanation">
+      <h2 class="headline">What is a Hackathon</h2>
```
Comment (clemenshelm) [inline source/index.html.erb:23]: Maybe you wanna mention the term "Hackathon" in the paragraph before, so the question is reasonable.

File: source/index.html.erb [hunk 2]
```diff
@@ -14,9 +14,31 @@ title: Foodhack Vienna | Coming Spring 2015
...
+    </div>
+
+    <div class="explanation">
+      <h2 class="headline">Project Info</h2>
+      <p>You don't have to know a lot about our current food waste problems. Over the next months we will be researching project ideas and we will provide information, data and sources.</p>
+      <p>So make sure you sign up for updates!</p>
+    </div>
+
+    <div class="explanation">
+      <h2 class="headline">Is There Anything Else</h2>
```
Comment (clemenshelm) [inline source/index.html.erb:38]: I personally would put a question mark after all of this interrogatory headlines, but I guess that's a matter of taste.

**Comment by clemenshelm** [comment]
I like it!

File: source/index.html.erb (hunk 2 above)
Comment (mraaroncruz) [inline source/index.html.erb:38]: it looks weird to me, i tried it. But I am willing to bend on this.
grrr Skitch
foo

Comment (clemenshelm) [inline source/index.html.erb:38]: No, nevermind. If it looks weird to you, leave it like it is.

File: source/index.html.erb (hunk 1 above)
Comment (mraaroncruz) [inline source/index.html.erb:23]: yes

**Comment by mraaroncruz** [comment]
@DominikGuzei there is a screenshot in my reply to clemens above

**Comment by DominikGuzei** [comment]
looks great! :+1: merge

**Pull request merged**